sdk/python/jobs/automl-standalone-jobs/automl-forecasting-github-dau/auto-ml-forecasting-github-dau.ipynb

```diff
@@ -0,0 +1,841 @@
+    "\n",
+    "|Property|Description|\n",
+    "|-|-|\n",
+    "|**allowed_training_algorithms**|The algorithms that will be allowed to train. All other models will be blocked.|\n",
+    "|**enable_dnn_training**|Enable Forecasting DNNs|"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# general job parameters\n",
+    "compute_name = \"github-cluster-sdkv2\"\n",
+    "max_trials = 5\n",
+    "exp_name = \"sdkv2-forecasting-github-dau\"\n",
+    "\n",
+    "target_column_name = \"count\"\n",
+    "forecast_horizon = 14\n",
+    "time_column_name = \"date\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.core.exceptions import ResourceNotFoundError\n",
+    "from azure.ai.ml.entities import AmlCompute\n",
+    "try:\n",
+    "    # Retrieve an already attached Azure Machine Learning Compute.\n",
+    "    compute = ml_client.compute.get(compute_name)\n",
+    "except ResourceNotFoundError as e:\n",
+    "    compute = AmlCompute(\n",
+    "        name=compute_name,\n",
+    "        size=\"STANDARD_DS12_V2\",\n",
+    "        type=\"amlcompute\",\n",
+    "        min_instances=0,\n",
+    "        max_instances=4,\n",
+    "        idle_time_before_scale_down=120,\n",
+    "    )\n",
+    "    poller = ml_client.begin_create_or_update(compute)\n",
+    "    poller.wait()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the AutoML forecasting job with the related factory-function.\n",
+    "\n",
+    "forecasting_job = automl.forecasting(\n",
+    "    compute=compute_name,\n",
+    "    experiment_name=exp_name,\n",
+    "    training_data=my_training_data_input,\n",
+    "    # validation_data = my_validation_data_input,\n",
+    "    target_column_name=target_column_name,\n",
+    "    primary_metric=\"NormalizedRootMeanSquaredError\",\n",
+    "    n_cross_validations=10,\n",
+    ")\n",
+    "\n",
+    "# Limits are all optional\n",
+    "forecasting_job.set_limits(\n",
+    "    timeout_minutes=120,\n",
+    "    trial_timeout_minutes=30,\n",
+    "    max_trials=max_trials,\n",
+    "    max_concurrent_trials=4,\n",
+    ")\n",
+    "\n",
+    "# Specialized properties for Time Series Forecasting training\n",
+    "forecasting_job.set_forecast_settings(\n",
+    "    time_column_name=time_column_name, forecast_horizon=forecast_horizon, frequency=\"D\"\n",
+    ")\n",
+    "\n",
+    "# Enable Dnn training and allow only TCNForecaster model\n",
+    "forecasting_job.set_training(\n",
+    "    allowed_training_algorithms=[\"TCNForecaster\"], enable_dnn_training=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.4 Run the Command\n",
```

Comment:
Please change this to `4.2 Train the AutoML model` to be consistent across notebooks.

Reply:
Changed

Reply:
The changes are not made. I still see the section titled `3.4 Run the Command`.